docs/design/network-discovery/switch-FSM.puml

```diff
@@ -28,8 +28,12 @@ endlegend
 
 [*] --> INIT
 
-INIT --> OFFLINE : history / setup port workers
 INIT --> SYNC : online
+INIT --> PENDING : history / setup port workers
```

Comment:
"setup port worker" is part of "apply history"

Reply:
changed, thanks